### PR TITLE
A new mission chain in Hai space

### PR DESCRIPTION
**Pull request by Arachi-Lover**
Adds some new missions in Hai space, where you assist a beginner Hai captain in getting some jobs.
Also adds four new variants for the "Nanachi 4" mission.

### CODE REVIEW

File: data/hai missions.txt [hunk 1]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Nanachi 1"
```
Comment (Amazinite) [inline data/hai missions.txt]: The first mission in a set should always be marked `minor`.

File: data/hai missions.txt [hunk 2]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+	name "Escort Nanachi to Cloudfire"
+	description "Teach Nanachi, a Hai who just bought her first ship, the basics of being a starship captain."
```
Comment (Amazinite) [inline data/hai missions.txt]: The description should probably name the destination.

File: data/hai missions.txt [hunk 3]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Nanachi 1"
+	name "Escort Nanachi to Cloudfire"
```
Comment (Amazinite) [inline data/hai missions.txt]: You can use `<planet>` here.

File: data/hai missions.txt [hunk 4]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+	description "Teach Nanachi, a Hai who just bought her first ship, the basics of being a starship captain."
+	source "Giverstone"
```
Comment (Amazinite) [inline data/hai missions.txt]: Why such a specific source? The conversation doesn't seem to point to the source needing to be so specific.

File: data/hai missions.txt [hunk 5]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+	destination "Cloudfire"
+	deadline
+	to offer
+		"net worth" > 20000000
+		"combat rating" > 1200
+		"random" > 85
```
Comment (Amazinite) [inline data/hai missions.txt]: `random` typically doesn't have quotes around it since it's a single-word keyword.

File: data/hai missions.txt [hunk 6]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
```
Comment (Amazinite) [inline data/hai missions.txt]: Nice name.

File: data/hai missions.txt [hunk 7]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+	on visit
```
Comment (Amazinite) [inline data/hai missions.txt]: `npc` and `on visit` should be below the `on offer` conversation for consistency with most other missions.

File: data/hai missions.txt [hunk 8]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+		dialog "You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system."
+	on offer
+		conversation
+			"As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding."
```
Comment (Amazinite) [inline data/hai missions.txt]: Use backticks, not quotes. Example:

```
`As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding.`
```

File: data/hai missions.txt [hunk 9]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+			choice
+				"(Approach the Hai.)"
+				"(Leave them alone.)"
```
Comment (Amazinite) [inline data/hai missions.txt]: Again: backticks, not quotes. Also, these should be tabbed in within the quotes.

```
choice
	`	(Approach the Hai.)`
	`	(Leave them alone.)`
```

File: data/hai missions.txt [hunk 10]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+					decline
+			`The hai is a bit startled, but quickly calms down, "Sorry, you caught me by surprise. I've just bought my first ship and need to find some jobs to do, but nobody I've talked to wants to trust a beginner captain. I just don't know what to do anymore, I've been working for so long to qualify for this loan, but I'm running out of time and I'll never be able to pay it like this."`
```
Comment (Amazinite) [inline data/hai missions.txt]: Again, tab in within the backticks. Also, Hai isn't capitalized here. There should also be a full stop after `but quickly calms down` 

```
`	The Hai is a bit startled, but quickly calms down. "Sorry, ..."`
```

File: data/hai missions.txt [hunk 11]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+				"(Wish her luck and leave.)"
+					decline
+			`Her ears twitch a bit, and her previously crestfallen eyes light up as she nearly jumps in joy at your offer, "Thank you so much captain! My name is Nanachi, I promise I won't cause any trouble at all, just a few quick jobs and I'm sure I'll be fine by myself!"`
+			`"Captain <first> <last>. Can you show me your ship so I can look for jobs it's better suited for?"`
+			`She hesitates a bit, but leads you to her ship: you were expecting a small Hai ship, but to your surprise she she shows you a rather old-looking Heavy Shuttle docked near your own ship.`
+			`"You took a loan to buy a human ship?" you ask.`
+			`She looks at the ground, as if ashamed of it, "It'd take way too long for me to qualify for a loan for even an Aphid, but I recently met a human merchant who wanted to sell his old ship; it was difficult but I managed to convince him to sell it to me instead of going to the shipyard office, I paid him its full price, but even for this one I had to take out the biggest loan I could. Its cargo hold isn't very impressive but it's got more bunks than our smallest ships, if that helps."`
+			choice
+				`(Continue.)`
```
Comment (Amazinite) [inline data/hai missions.txt]: If you're adding a choice that is nothing but (Continue.) then you're probably doing something wrong.

File: data/hai missions.txt [hunk 12]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+			`"My friend Nanachi here could deliver it for you. We were heading there anyway, a few extra crates won't make a difference," you say, trying to sound as convincing as possible.`
+			`"Look kid, Unfettered attacks make it so they need a lot of medical supplies in Cloudfire, and this here is the best medical equipment Hai space has to offer, these few crates could save many military lives and I don't think I should be trusting something like this to a beginner," he says.`
+			`If you were the client for this job you certainly wouldn't want a beginner going through such a dangerous territory with such valuable cargo, but he seems desperate, and seeing as the spaceport is deserted he doesn't have many other captains to look for help, perhaps you could convince him.`
+			choice
+				`"Beginner or not, I'll be escorting her there. I'm pretty confident on my skills and my ship is nothing to scoff at, so I'll keep her, and the cargo, safe."`
+					goto safe
+				`"Don't worry, the planet is one jump away, what could possibly go wrong?"`
+					goto wrong
+				"(Make up stories about Nanachi's experience.)"
```
Comment (Amazinite) [inline data/hai missions.txt]: Always use backticks, not just when what's inside them requires quotation marks.

File: data/hai missions.txt [hunk 13]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+
+
+mission "Nanachi 4"
+	name "Protect Nanachi"
+	description "Keep Nanachi's ship safe from a pirate fleet that has attacked a convoy heading to Hai space."
+	source "Allhome"
+	stopover "Prime"
+	deadline 14
+	to offer
+			has "Nanachi 3: done"
```
Comment (Amazinite) [inline data/hai missions.txt]: This is tabbed in too far.

File: data/hai missions.txt [hunk 14]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+	npc
+		personality nemesis waiting
+		government "Pirate"
+		system "Betelgeuse"
+		ship "Marauder Quicksilver (Hai)" "Hazelnut's Bane"
+	npc
+		personality nemesis waiting
+		government "Pirate"
+		system "Betelgeuse"
+		ship "Marauder Bounder (Hai)" "Rodent Hunter"
```
Comment (Amazinite) [inline data/hai missions.txt]: Do these all need to be separate `npc`s? Can you not just put them all under one block?

File: data/hai missions.txt [hunk 15]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+
+mission "Nanachi: Payment"
+	landing
+	source
+		government "Hai"
+		not planet "Darkwaste"
+	to offer
+		has "event: Nanachi: payment"
+	on decline
+		event "Nanachi: Update" 720
```
Comment (Amazinite) [inline data/hai missions.txt]: This can just go in the `on offer` alongside the payment.

File: data/hai missions.txt [hunk 16]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+mission "Nanachi Meeting"
+	landing
+	source
+		government "Hai"
+		not planet "Darkwaste"
+	to offer
+		has "event: Nanachi: Update"
+		"random" > 30
+	on offer
+		conversation
```
Comment (Amazinite) [inline data/hai missions.txt]: This conversation doesn't have any endpoint. It should probably have `decline`.

File: data/hai missions.txt [hunk 17]
```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
...
+			choice
+				`(Continue.)`
+			`Turns out this isn't as easy as you thought it'd be, the spaceport is very empty, save for the occasional worker or drunken crewmember, plus you see some workers pointing at Nanachi and murmuring, it seems her inability to get a job has given her a bad reputation here. Eventually, as you two begin scouting the edge of the docking area, you spot a troubled merchant near several crates, you approach him and he explains that his ship had some malfunctions and he cannot deliver this advanced surgical equipment to Cloudfire.`
+			`"This is the worse! One jump away from the planet and the navigation system suddenly decides to go on strike! There's no way I can make it before the deadline!"`
```
Comment (Darcman99) [inline data/hai missions.txt]: "worse" -> "worst"?

**Comment by Pointedstick** [comment]
Did you mean to close this?

**Pull request closed without merging**